.travis.yml

```diff
@@ -0,0 +1,20 @@
+langauage: node
+
+node_js:
+  - node
+
+env:
+  - NODE_ENV=test
+
+services:
+  - sqlite3
+
+install:
+  - npm install
+
+script:
+  - npm test
+
+notifications:
+  slack:
+    secure: C39KywRCO97fSN5UqAcpamWH724a3Kv97176npKpQ38b8gFR0CGxUrtsi/QaK4rwVgyMJIo/vVuirgem6szlHf52jctvJIExukXLQGJDWUB+Lhr+gOmTECutcaKhOfLod790eY+L06FJCWYSfMiFLbB+ET/0UNJgtafWqi5aqZB1zqv1e3s2L+cGgF8pZ4XvboY5bC4y/Sfn7QYj/0L6uht0zlhItqd5GgMwNr2ygON6HbPHDnV0vdb/6Th+fFzU3GFe79Vwa2l3kowZVxIEtxqBoIgS7szoHUL+cq6xWQIrSG0lKg5i/XpQ/5wxaQXT9o8yB+owJRbWZCef7UVvJkssJeBeSb1tyij5ETPp3ryFGEuVZg2LHKiwa2m6OiRbf5ieTKq0gXD8ZoHOEa9SOtyvPpCc4ogurJ6RLY5AbYJxHfV8x4pANxzav75dxVa3iDXht4FJePoYDel/nxIAEgPQ99mMBnoGK8aVKZFGT9QBuOfmRG3j4BIDbbEgrtpAHWLWas9QuqYN4U1Pzq9JcgeOwUzoAGo3rzTT9wlEYjxODHQmkFI+juV30Q45mEekvNs58ws6crPrf32N3xW9o8XU9Gl4hHZZavsRcbZ+rZ5D4JsR2pvvAHqjbVMHDJdanpRugEgUB1sleutHo5jZ0DeYKq5S3qobqD9SD7fEVC8=
```

Comment:
Find a way to put this token in .env. I told you to talk to Ndifreke

Reply:
I already message him. He's yet to reply to me

Reply:
At your request to put the token in .env - I got to realise that Travis does not support that feature yet according to (travis-ci/travis-ci#7201) and (travis-ci/travis-ci#6387) issues, rather the token can be secured by running Travis encrypt command on the generated token, thereby producing an encrypted string of the token which can now be passed into notifications as I did above.